src/typemaps.jl

```diff
@@ -9,118 +9,381 @@ It is expected (but might not be guaranteed) that these are the same across vers
+    :pg_stat_database => 11704,
+    :pg_stat_database_conflicts => 11707,
+    :pg_stat_user_functions => 11710,
+    :pg_stat_xact_user_functions => 11714,
+    :pg_stat_archiver => 11718,
+    :pg_stat_bgwriter => 11721,
+    :pg_stat_progress_vacuum => 11724,
+    :pg_user_mappings => 11728,
+    :pg_replication_origin_status => 11732,
+    :cardinal_number => 12280,
+    :character_data => 12282,
+    :sql_identifier => 12283,
+    :information_schema_catalog_name => 12285,
+    :time_stamp => 12287,
+    :yes_or_no => 12288,
+    :applicable_roles => 12291,
+    :administrable_role_authorizations => 12295,
+    :attributes => 12298,
+    :character_sets => 12302,
+    :check_constraint_routine_usage => 12306,
+    :check_constraints => 12310,
+    :collations => 12314,
+    :collation_character_set_applicability => 12318,
+    :column_domain_usage => 12322,
+    :column_privileges => 12326,
+    :column_udt_usage => 12330,
+    :columns => 12334,
+    :constraint_column_usage => 12338,
+    :constraint_table_usage => 12342,
+    :domain_constraints => 12346,
+    :domain_udt_usage => 12350,
+    :domains => 12354,
+    :enabled_roles => 12358,
+    :key_column_usage => 12361,
+    :parameters => 12365,
+    :referential_constraints => 12369,
+    :role_column_grants => 12373,
+    :routine_privileges => 12376,
+    :role_routine_grants => 12380,
+    :routines => 12383,
+    :schemata => 12387,
+    :sequences => 12390,
+    :sql_features => 12394,
+    :pg_toast_12393 => 12396,
+    :sql_implementation_info => 12399,
+    :pg_toast_12398 => 12401,
+    :sql_languages => 12404,
+    :pg_toast_12403 => 12406,
+    :sql_packages => 12409,
+    :pg_toast_12408 => 12411,
+    :sql_parts => 12414,
+    :pg_toast_12413 => 12416,
+    :sql_sizing => 12419,
+    :pg_toast_12418 => 12421,
+    :sql_sizing_profiles => 12424,
+    :pg_toast_12423 => 12426,
+    :table_constraints => 12429,
+    :table_privileges => 12433,
+    :role_table_grants => 12437,
+    :tables => 12440,
+    :transforms => 12444,
+    :triggered_update_columns => 12448,
+    :triggers => 12452,
+    :udt_privileges => 12456,
+    :role_udt_grants => 12460,
+    :usage_privileges => 12463,
+    :role_usage_grants => 12467,
+    :user_defined_types => 12470,
+    :view_column_usage => 12474,
+    :view_routine_usage => 12478,
+    :view_table_usage => 12482,
+    :views => 12486,
+    :data_type_privileges => 12490,
+    :element_types => 12494,
+    :_pg_foreign_table_columns => 12498,
+    :column_options => 12502,
+    :_pg_foreign_data_wrappers => 12505,
+    :foreign_data_wrapper_options => 12508,
+    :foreign_data_wrappers => 12511,
+    :_pg_foreign_servers => 12514,
+    :foreign_server_options => 12518,
+    :foreign_servers => 12521,
+    :_pg_foreign_tables => 12524,
+    :foreign_table_options => 12528,
+    :foreign_tables => 12531,
+    :_pg_user_mappings => 12534,
+    :user_mapping_options => 12538,
+    :user_mappings => 12542,
```

Comment:
I'd want to ignore this change as well

Reply:
i think maybe in this case we just don't want to format this block at all? e.g. we could add https://domluna.github.io/JuliaFormatter.jl/dev/skipping_formatting/

Reply:
Yes, perfect